packages/dialtone-tokens/build-sd-transforms.js

```diff
@@ -43,22 +43,36 @@ export async function run () {
     // use $metadata to iterate through the selected token sets
     // as this contains the correct token set order. The $themes
     // file does not.
-    const source = $metadata
+
+    // includes are treated as SOURCE tokens, meaning they
+    // will be used as a reference but will not be output.
+    const include = $metadata
```

Comment:
Does this mean that we can no longer use tokens like `--dt-color-black-700`? Should these be updated in the product?

Reply:
nope they can still be used. they are still included in the base-tokens file. They are just not included in the brand tokens files where we want to output semantic tokens only